modules/library/render/src/main/java/org/geotools/renderer/lite/StreamingRenderer.java

```diff
@@ -1193,7 +1174,7 @@ Query getStyleQuery(FeatureSource<FeatureType, Feature> source,
      * @param styles
      * @return
      */
-    private SortBy[] getSortByFromLiteStyles(LiteFeatureTypeStyle[] styles) {
+    private SortBy[] getSortByFromLiteStyles(List<LiteFeatureTypeStyle> styles) {
         for (LiteFeatureTypeStyle fts : styles) {
             if (fts.sortBy != null) {
                 return fts.sortBy;
```

Comment:
I may be missing the logic here, are we just returning the value from the first FeatureTypestyle to have a sortBy (like do we have a danger of missing information if two FeatureTypeStyles contain sortBy information.

If this is intensional I expect method name, or javadoc can clarify.